.agents/qa-bugbot/README.md

```
@@ -0,0 +1,83 @@
+
+### 2. CLI (single PR, manual)
+
+```bash
+TARGET_PR_URL=https://github.com/org/repo/pull/99 npm start
+```
+
+Or set `TARGET_PR_NUMBER=99` with `REPO_URL` in `.env`.
+
+## Chain
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)